src/systemdhook.c

```diff
@@ -88,6 +88,26 @@ DEFINE_CLEANUP_FUNC(yajl_val, yajl_tree_free)
 #define CGROUP_ROOT "/sys/fs/cgroup"
 #define CGROUP_SYSTEMD CGROUP_ROOT"/systemd"
 
+/* 
+   Values in the array as found in /proc/self/cgroup inside
+   of the container and are used for systemd processing.
+*/
+const char* const sysdHierArray[] = {
```

Comment:
Any way we can generate this on the fly?  Does not seem very future proof?

Reply:
Yeah I was contemplating that last night after I pushed this.  Will see what I can massage.  The issue I'm having atm is finding the unified hierarchy, it's not in the file that we're currently reading.  I'll also have to change the handling of the file that we read for the hierarchies, should be doable.

Reply:
The latest version generates on the fly, I've removed the two arrays that I had with the directory names in them.